edgebuilder-install.sh

```diff
@@ -468,6 +471,101 @@ install_cli_rpm()
   fi
 }
 
+# Uninstall the Server components
+uninstall_server()
+{
+    if dpkg -s edgebuilder-server; then
+
+        sudo rm -rf /opt/edgebuilder/server/vault
+        # attempt autoremove
+        if sudo apt autoremove -qq edgebuilder-server -y ;then
+            echo "Successfully autoremoved server components"
+        else
+            echo "ERROR: an issue running autoremove server components"
+            exit 1
+        fi
+
+        # attempt purge
+        if sudo apt-get -qq purge edgebuilder-server -y ;then
+            echo "Successfully purged Server Components"
+        else
+            echo "ERROR: an issue running autoremove Server Components"
+            exit 1
+        fi
+
+        # Successfully installed, exit
+        echo "Server Components Uninstalled"
+        exit 0
+    else
+        # package not currently installed, so exit
+        echo "Edgebuilder-server NOT currently installed"
```

Comment:
Looks like you're referring to the package name here. If so I think it's best to called it "edgebuilder-server"